tests/unit/test_neo4j_queries.py

```diff
@@ -23,7 +23,7 @@ def test_vector_search_basic() -> None:
     expected = (
         "CALL db.index.vector.queryNodes($vector_index_name, $top_k, $query_vector) "
         "YIELD node, score "
-        "RETURN node { .*, `None`: null } AS node, labels(node) AS nodeLabels, elementId(node) AS id, score"
+        "RETURN node { .*, `None`: null } AS node, labels(node) AS nodeLabels, elementId(node) AS nodeId, elementId(node) AS id, score"
```

Comment:
I think we need clear rules for when we use "id" as a variable in the package. Here "id" is the elementId ID but there are other places where we use "id" to refer to a LLM created ID. Maybe we should use "id" as a variable name for user or LLM created IDs and "elemID" for element IDs

Reply:
I would suggest that we use `element_id` for both node and relationship ids, because that's what they are.
It will also slowly teach people the terminology used in Neo4j.